vsphere-RHEL-manager-blueprint-inputs.yaml

```diff
@@ -26,31 +26,25 @@
 #management_network_cidr: ""
 #management_network_gateway: ""
 #management_network_ip: ""
-#management_network_switch_distributed: true
+#management_network_switch_distributed: false
 
 # Network Settings for external network
 #external_network_use_dhcp: true
 #external_network_cidr: ""
 #external_network_gateway: ""
 #external_network_ip: ""
-#external_network_switch_distributed: true
-
-#############################
-# Provider specific Inputs
-#############################
+#external_network_switch_distributed: false
 
 # SSH user used to connect to the manager
 #ssh_user: "root"
 
 # SSH key path used to connect to the manager
 #ssh_key_filename: ~/.ssh/centos_key.pem
+# SSH key path used to connect from the manager to nodes
 #agent_private_key_path: ~/.ssh/cloudify-agent-kp.pem
 
-# SSH port used to connect to the manager
-#ssh_port: 22
-
 # This is the user with which the Manager will try to connect to the application hosts.
-#agents_user: ''
+#agents_user: 'ubuntu'
```

Comment:
I think this should actually not have a default as it very much depends on the user's stack.